sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/models/RoleScope.java

```diff
@@ -0,0 +1,30 @@
+package com.azure.security.keyvault.administration.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.net.URI;
+
+public final class RoleScope extends ExpandableStringEnum<RoleScope> {
```

Comment:
Should be `RoleAssignmentScope`.

Reply:
Please refer to my other comment on `RoleScope`.